VOM_Fortran/VOM-code/transpmodel.f90

```diff
@@ -1271,14 +1269,12 @@ subroutine vom_hourly_init ()
      end do
 
 !     * (Out[310], derived from (3.26)) Temperature dependence of Jmax
-      jmaxg_h(:) = (p_E ** ((i_ha * (-25.d0 + tair_h(th_)) * (-273.d0  &
-     &           + topt_ + 273.d0 * p_R_ * topt_)) / ((25.d0 + 273.d0  &
-     &           * p_R_ * topt_) * (tair_h(th_) + 273.d0 * p_R_        &
-     &           * topt_))) * ((-1.d0 + p_E ** (-(i_hd * (-298.d0      &
-     &           + topt_)) / (25.d0 + 273.d0 * p_R_ * topt_))) * i_ha  &
-     &           + i_hd) * jmax25g_d(:)) / ((-1.d0 + p_E ** ((i_hd     &
-     &           * (273.d0 + tair_h(th_) - topt_)) / (tair_h(th_)      &
-     &           + 273.d0 * p_R_ * topt_))) * i_ha + i_hd)
+      jmaxg_h(:) = (p_E ** ((i_ha * (-25.d0 + tair_h(th_))) / ((25.d0 + 273.d0)  &
+        &           * p_R_ * (tair_h(th_) + 273.d0) )) * ((-1.d0 + p_E ** (-(i_hd * (-298.d0      &
```

Comment:
This line is too long. I would re-write as:
```
     jmaxg_h(:) = (p_E ** ((i_ha * (-25.d0 + tair_h(th_)))            & 
    &           / ((25.d0 + 273.d0) * p_R_ * (tair_h(th_) + 273.d0))) &
    &           * ((-1.d0 + p_E ** (-(i_hd * (-298.d0                 &
```

Reply:
Yes, agree, that looks better... Made changes accordingly.